spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/AnalyzeTableSparkAction.java

```diff
@@ -47,13 +55,20 @@ public class AnalyzeTableSparkAction extends BaseSparkAction<AnalyzeTableSparkAc
   private static final Logger LOG = LoggerFactory.getLogger(AnalyzeTableSparkAction.class);
 
   private final Table table;
-  private Set<String> columns = ImmutableSet.of();
-  private Set<String> types = StandardBlobTypes.blobTypes();
+  private final Set<String> supportedBlobTypes =
+      ImmutableSet.of(StandardBlobTypes.APACHE_DATASKETCHES_THETA_V1);
+  private Set<String> columns;
+  private Set<String> blobTypesToAnalyze = supportedBlobTypes;
```

Comment:
What about combining these into a list of (blob type, columns) pairs?
This might be necessary when we add support for new blob types.
see https://github.com/apache/iceberg/pull/10288/files#r1639547902